api_views/main.py

```diff
@@ -17,3 +17,10 @@ def basic():
                     'in APIs but it can also be used in learning/teaching purposes.", "vulnerable":' + "{}".format(vuln) + "}"
     response = Response(response_text, 200, mimetype='application/json')
     return response
+
+def basic2():
+    response_text = '{ "message": "VAmPI the Vulnerable API", "help": "VAmPI is a vulnerable on purpose API. It was ' \
+                    'created in order to evaluate the efficiency of third party tools in identifying vulnerabilities ' \
+                    'in APIs but it can also be used in learning/teaching purposes.", "vulnerable":' + "{}".format(vuln) + "}"
+    response = Response(response_text, 400, mimetype='application/json')
+    return response
```

Comment:
Response uses 400 status code but appears to be successful response - should use 200